docs/tutorial_custom.rst

```diff
@@ -58,11 +58,12 @@ Multiple return types are supported:
 **Type** 	        **Notes**
 ---------------     -----------------------------------
   tuple             (data_name, data_to_be_added)
-  dict              Data to be added keyed by data_name
+  dict              Data to be added keyed under 'data'
```

Comment:
> Data to be added keyed under 'data'

What does this mean?

Reply:
The return dict should be {'data': data_to_be_added, ...}.

Reply:
Ok.  I think what you need to do here is spell out the whole dictionary, since this is one of the key places people (me in the past) look to see how the input should be structured.  At a minimum, point to a place where the whole dictionary is described.

Reply:
I agree. My plan was to tackle that portion in #635 once we settled on what the add method return format should be.
The current format doesn't leave much room for exposition so was planning on improving that as well.

Reply:
Ok!  I'd recommend adding a task to that issue that points to this file and line so that it's easy to remember to fix it in the future.